<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ikoOSKAR::UI::StudentEditorUi</class>
 <widget class="QDialog" name="ikoOSKAR::UI::StudentEditorUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="lblHeader">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Öğrenci [Ekle|Bilgilerini Düzenle]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>40</number>
      </property>
      <property name="verticalSpacing">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblId">
        <property name="text">
         <string>Okul No:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblFirstName">
        <property name="text">
         <string>Adı:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lnFirstName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblLastName">
        <property name="text">
         <string>Soyadı:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lnLastName"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblGrade">
        <property name="text">
         <string>Sınıfı:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spnGrade">
        <property name="minimum">
         <number>9</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblSection">
        <property name="text">
         <string>Şubesi:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmbSection">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="maxVisibleItems">
         <number>9</number>
        </property>
        <item>
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>E</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>J</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>K</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>L</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>N</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>O</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>P</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>R</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>V</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>W</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spnId">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>2147483646</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ikoOSKAR::UI::StudentEditorUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ikoOSKAR::UI::StudentEditorUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
